docs/sphinx/using/hardware.rst

```diff
@@ -0,0 +1,170 @@
+CUDA Quantum Hardware Backends
+*********************************
+
+The hardware vendors currently available in CUDA Quantum are as follows.
+
+IonQ
+==================================
+
+Setting Credentials
+```````````````````
+
+Programmers of CUDA Quantum may access the `IonQ Quantum Cloud
+<https://cloud.ionq.com/>`_ from either C++ or Python. Simply generate
+an API key from your `IonQ account <https://cloud.ionq.com/>`_ and export
+it as an environment variable:
+
+.. code:: bash
+
+  export IONQ_API_KEY="ionq_generated_api_key"
+
+
+C++
+```
+
+For developers in C++, you can indicate to ``nvq++`` that your quantum
+kernels will be executed in the IonQ Cloud via the ``--target`` flag.
+
+.. code:: bash
+
+    nvq++ --target ionq src.cpp ...
+
+This will take the API key and handle all authentication with, and submission to,
+the IonQ QPU.
+
+Note: A "target" in :code:`cudaq` refers to a quantum compute provider, such as :code:`ionq`.
+However, IonQ's documentation uses the term "target" to refer to specific QPU's themselves.
+
+Specifying the QPU
+'''''''''''''''''''
+
+At this time, programmer control over the IonQ QPU is under construction.
+
+
+Example
+.......
+
+.. literalinclude:: ../examples/cpp/ionq
+    :language: cpp
+
+
+
+Python
+```````
+
+For python developers, the target may be controlled with the ``cudaq::set_target()``
+function. This is functionally equivalent to the ``nvq++`` target,
+and will handle the submission of all quantum kernels to IonQ.
+
+.. code:: python
+
+    cudaq.set_target('ionq')
+
+Note: A "target" in :code:`cudaq` refers to a quantum compute provider, such as :code:`ionq`.
+However, IonQ's documentation uses the term "target" to refer to specific QPU's themselves.
+
+Specifying the QPU
+'''''''''''''''''''
+
+At this time, programmer control over the IonQ QPU is under construction.
```

Comment:
We need to revise that before merging. It should be possible to set the qpu. You should be able to confirm that it works by trying to submit to a hardware backend. Your job will get rejected since you don't have any credit for it.